tests/installation/partitioning_raid.pm

```diff
@@ -162,31 +166,38 @@ sub run() {
         assert_screen 'partitioning-size';
         send_key 'ctrl-a';
         type_string "200 MB";
+        assert_screen 'partitioning_raid-custom-size-200MB';
         send_key 'alt-n';
         assert_screen 'partition-role';
-        send_key "alt-a";          # Raw Volume
+        send_key "alt-a";
+        assert_screen 'partitioning_raid-partition_role_raw_volume';
         send_key 'alt-n';
         assert_screen 'partition-format';
         send_key 'alt-d';
+        assert_screen 'partitioning_raid-format_noformat';
         send_key 'alt-i';
+        assert_screen 'partitioning_raid-file_system_id-selected';
         send_key_until_needlematch 'filesystem-prep', 'down';
-        send_key 'ret';
         send_key 'alt-f';
         assert_screen 'custompart';
         send_key 'alt-s';
         send_key 'right';
```

Comment:
same as above, one keyboard shortcut that should cause some action and then it immediately presses the next key without waiting for the SUT to finish processing the previous request